tests/in/access.param.ron

```diff
@@ -1,7 +1,9 @@
 (
-	spv_version: (1, 1),
-	spv_debug: true,
-	spv_adjust_coordinate_space: false,
+	spv: (
+		version: (1, 1),
+		debug: true,
+		adjust_coordinate_space: false,
```

Comment:
We should omit the values that are the default, right?

Reply:
we *could* but not necessarily should :)